cmd/daprd/main.go

```diff
@@ -152,6 +152,13 @@ var (
 )
 
 func main() {
+	// set GOMAXPROCS
+	undo, err := maxprocs.Set()
+	defer undo()
+	if err != nil {
+		log.Fatal("failed to set GOMAXPROCS: %v", err)
+	}
+
```

Comment:
Thanks for fixing it. Generally looks good to me, just `maxprocs.Set()` I think is enough, undo and error handling are not necessary. Besides, better move it to `init`.

```go
func init() {
	maxprocs.Set()
}
```

Reply:
Done.